fixtures/artifactor_plugin.py

```diff
@@ -249,14 +249,13 @@ def pytest_unconfigure(config):
 def shutdown(config):
     with lock:
         proc = config._art_proc
-        if proc:
+        if proc.returncode is None:
```

Comment:
i'd use `getattr(proc, 'returncode', None) is None`

the logic below is also missing something, 
the `proc.wait` that is nested in here only makes sense when the process is not yet terminated and needs to be told